lib/utils/globals.dart

```diff
@@ -1,8 +1,17 @@
 import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
 import 'package:smart_chef/utils/colors.dart';
 
 const double bottomIconSize = 55;
 const double bottomRowHeight = 90;
+const double topBarIconSize = 28;
+
+bool ingredientPage = true;
+bool recipePage = false;
+bool shoppingCartPage = false;
+bool userProfilePage = false;
```

Comment:
What are these bools used for?

Reply:
i forgot to get rid of them. was gonna try to break the repetition of the bottom bar buttons but it ended up not working out.